Documentation/contributing/development/dev_setup.rst

```diff
@@ -339,6 +339,20 @@ to enable NFS.
    affected. A workaround for now is to pass ``ibt=off`` to the kernel
    command line.
 
+.. note::
+
+   VirtualBox for Ubuntu desktop might have network issues after
+   suspending and resuming the host OS (typically by closing and
+   re-opening the laptop lid). If the ``cilium status`` keeps showing
+   unreachable from nodes but reachable from endpoints, you could
+   hit this. Run the following code on each VM to rebuild routing
+   and neighbor entries:
+
+   .. code-block:: shell-session
+
+      $ ip -6 r a fd00::/16 dev enp0s9
+      $ ip n r $REMOTE_HOST_IPV6 dev enp0s9 lladdr $REMOTE_HOST_MAC nud reachable
```

Comment:
Thanks!

Any chance you could come up with an `ip` command to set `REMOTE_HOST_IPV6` and `REMOTE_HOST_MAC` before running this last command (`REMOTE_HOST="$(ip ...)"`), so that the user does not have to look it up?

Reply:
Sure thing! Just updated, please check.